backend/pyproject.toml

```diff
@@ -57,7 +57,7 @@ orcid = "==1.0.3"
 
 node-semver = "<0.2.0,>=0.1.1"
 pybtex = "*"
-cryptography = "<2.6"
+cryptography = "^36.0.2"
```

Comment:
36?

Reply:
yes, it seems that they put a wrong numbers by mistake https://pypi.org/project/cryptography/3.0/#history

Reply:
🤕